### DISCUSSION TITLE
Latest Episodes missing thumbnails

### DISCUSSION THREAD

**Discussion by sttrebo** [SamTV12345/PodFetch#1081] [Q&A]
I just noticed that on my 'Latest Episodes' page, none of the episode thumbnails are showing.  don't know if this happened last night or a few days ago since i haven't looked in a little while (sorry for being vague).  but in my browser console see this error message     
        
       Failed to load resource: the server responded with a status of 401 ()

I have not made any changes since I installed Podfetch (installed on 12/6) and created an admin user so i don't think anything has changed on my end.  Is this a mis-configuration on my end or am I missing something?

when i select the link in the error message, I get 'unauthorized'

![Screenshot 2024-12-25 102550](https://github.com/user-attachments/assets/c41b8ad9-0a90-4889-8882-5f06ca7aa9a5)

**Comment by SamTV12345** [comment]
I think they should be back. There was an oversight in the exporting of podcasts

**Reply by sttrebo** [reply]
just did a fresh install and added in my pods (admin account).  still get the same issue (above).
here's a screenshot of my sys config screen.
![Screenshot 2025-01-19 073648](https://github.com/user-attachments/assets/6d5ad98b-2284-4bb8-aea5-c381c13a5024)
no other issues so this is more of an annoyance and not a problem.

**Reply by SamTV12345** [reply]
If you use the admin account with basic auth you also need to set the variable API_KEY in order to access the podcast images. Probably you don't have this enabled and the server denies your request. Let me know if that fixes your issue :)

**Reply by sttrebo** [reply]
bingo, that was it.  thanks for your help.  and sorry about the newbie questions.

**Reply by SamTV12345** [reply]
No worries. I should add more notifications and warnings to avoid these pitfalls. Will do that next 😄.
